src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStoreCreation.swift

```diff
@@ -0,0 +1,126 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import FMDB
+
+extension ContactDiaryStore {
+
+	// MARK: - Init
+
+	convenience init?(url: URL) {
+
+		guard let databaseQueue = FMDatabaseQueue(path: url.path) else {
+			Log.error("[ContactDiaryStore] Failed to create FMDatabaseQueue.", log: .localData)
+			return nil
+		}
+
+		let latestDBVersion = 4
+		let schema = ContactDiaryStoreSchemaV4(
+			databaseQueue: databaseQueue
+		)
+
+		let migrations: [Migration] = [
+			ContactDiaryMigration1To2(databaseQueue: databaseQueue),
+			ContactDiaryMigration2To3(databaseQueue: databaseQueue),
+			ContactDiaryMigration3To4(databaseQueue: databaseQueue)
+		]
+		let migrator = SerialDatabaseQueueMigrator(
+			queue: databaseQueue,
+			latestVersion: latestDBVersion,
+			migrations: migrations
+		)
+
+		self.init(
+			databaseQueue: databaseQueue,
+			schema: schema,
+			key: ContactDiaryStore.encryptionKey,
+			migrator: migrator
+		)
+	}
+
+	// MARK: - Internal
+
+	static func make(url: URL? = nil) -> ContactDiaryStore {
+		let storeURL: URL
+
+		if let url = url {
+			storeURL = url
+		} else {
+			storeURL = ContactDiaryStore.storeURL
+		}
```

Comment:
```suggestion
			let storeURL = url ?? ContactDiaryStore.storeURL
```
(supposed to be aligned correctly)